src/visualMarkers/main.ts

```diff
@@ -49,6 +54,14 @@ async function handleMarkChangeEvent(event: MarkChangeEvent) {
 
       await smoke(addedMark.position, color)
 
+      // if duration is set by the user, deploy smoke in interval
+      if (command.duration != undefined) {
+        const timerID = setInterval(
```

Comment:
Unfortunately, you need to do something extra here to make this safe.

currently, yads can disconnect and reconnect to the dcs server multiple times during a game session, and when it does we lose all context and the ability to call dcs-grpc api's.

today this is solved because every module has a teardown fn, that cleans up after itself.
We will need to add this to the cleanup/teardown.

If there are any existing setInterval when we teardown, we'll have to `clearInterval` on them all.

if you want smoke timings to work across restarts/teardowns you will have to store them in the db